forte/data/types.py

```diff
@@ -24,9 +24,4 @@
 
 DataRequest = Dict[Type[Entry], Union[Dict, List]]
 
-try:
-    import torch
-
-    MatrixLike = Union[torch.TensorType, np.ndarray, List]
-except ImportError:
-    MatrixLike = Union[np.ndarray, List]  # type: ignore
+MatrixLike = Union[np.ndarray, List]
```

Comment:
Hmm, why is this file changed? Hopefully a tutorial won't change anything in the library itself.

Reply:
Does this break this PR? This issue should probably be resolved in another PR with more investigation, not piggybacking this PR.

The current fix is not desired. We want to make MatrixLike to include pytorch tensor types if torch is available. We cannot just comment it out.

Reply:
I have added the code back. And it doesn't break the PR this time. so we can ignore this file change now.

Reply:
ok, shall we create an issue describing what you think it's wrong?